zerver/openapi/zulip.yaml

```diff
@@ -22195,6 +22218,12 @@ components:
             has older history that can be accessed.
 
             Is `null` for channels with no message history.
+        is_recently_active:
+          type: boolean
+          description: |
+            Whether the channel has any new messages sent to it recently.
+
+            **Changes**: New in Zulip 9.0.
```

Comment:
It looks like there are two properties with this name being added, with somewhat different descriptions. Do the facts about them differ, or should the descriptions be the same?

Reply:
Thanks for noticing. Updated this match the other description.

```quote
Whether the channel has recent message activity. Clients should use this to implement [sorting inactive channels to the bottom](/help/manage-inactive-channels).

**Changes**: New in Zulip 10.0 (feature level 301)
```